book/theme/index.hbs

```diff
@@ -0,0 +1,365 @@
+<!DOCTYPE HTML>
```

Comment:
How was this generated? How do we keep it up to date in the future?

Reply:
> How was this generated?

I copied <https://github.com/rust-lang/mdBook/blob/master/src/theme/index.hbs>, and edited inside nav#sidebar.

> How do we keep it up to date in the future?

I guess we can write a reminder somewhere to look into <https://github.com/rust-lang/mdBook/commits/master/src/theme/index.hbs> before tagging a new version?

Reply:
Can we have a script to do it easily instead?

Reply:
Added.